src/remoteplayer.h

```diff
@@ -101,6 +101,9 @@ class RemotePlayer : public Player
 		*clouds = m_sky_clouds;
 	}
 
+	void setFov(const f32 fov) { m_fov = fov; }
+	const f32 getFov() const { return m_fov; }
```

Comment:
These don't seem to be necessary. Just pass the FOV value to the network packet sending function (like breath, HP).

Reply:
Only server->client FOV sending has been implemented, which is why I added these to `RemotePlayer` to keep track of the player's FOV.

Reply:
Adding a client->server FOV sending will require another server->client "request" packet, which is why it was not implemented